### PR TITLE
Updated data for institute, academics, people, admission

### PR DESCRIPTION
**Pull request by thisisnitish**
Updated all the data from the original website to the current project.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,17 +9,17 @@
...
     },
     "@babel/compat-data": {
       "version": "7.11.0",
       "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.11.0.tgz",
       "integrity": "sha512-TPSvJfv73ng0pfnEOh17bYMPQbI95+nGWc71Ss4vZdRBHTDqmM9Z8ZV4rYz8Ks7sfzc95n30k6ODIq5UGnXcYQ==",
       "requires": {
-        "browserslist": "4.14.0",
-        "invariant": "2.2.4",
-        "semver": "5.7.1"
+        "browserslist": "^4.12.0",
```
Comment (fahad-israr) [inline package-lock.json:20]: Version downgrade. why?
see at all places.

File: src/components/person_card/index.js [hunk 2]
```diff
@@ -25,37 +25,56 @@ const PersonCard = (props) => {
 
   return (
     <>
-    <div className="card">
-    <div className="top">
-       <div className="text">
-         <a className="name" href={`/faculty_detail/${props.dept}/${props.deptID}`}>{props.name}</a>
-         <h4>{props.designation}</h4>
-        </div>
+      <div className="card">
```
Comment (fahad-israr) [inline src/components/person_card/index.js:28]: Why modifying this code when no changes?

File: src/components/person_card/index.js [hunk 3]
```diff
@@ -25,37 +25,56 @@ const PersonCard = (props) => {
...
+              >
+                {props.name}
+              </a>
+            )}
+            <h4>{props.designation}</h4>
+          </div>
 
-        <div className="circle-img">
-         <img  src={require(`../../images/people/${props.src_type}/${props.src}`)} alt="avatar_img" />
+          <div className="circle-img">
```
Comment (fahad-israr) [inline src/components/person_card/index.js:49]: here too...unncessary modification

Comment (fahad-israr) [reply src/components/person_card/index.js:49]: *unnecessary

**Review by fahad-israr — COMMENTED** [review]
- Unnecessary version downgrades everywhere
- Other minor changes as  I have left out comment.
- NEWS, EVENTS and Notices need to have  exact same contest as the main website.
- Faculty page (i.e on Clicking Faculty Name) Must render a page as you had built this earlier.

File: src/components/person_card/index.js (hunk 2 above)
Comment (thisisnitish) [reply src/components/person_card/index.js:28]: >* The thing is there is indentation happens when I save any file. If you see nothing much is changed I only updated the informations
>* The one thing I've changed is when you have link on the faculty cards and when you will click it you will see details but what about staff we can't see their details, so there was a bug in that you just check with current version, when you will the click the staff name you will see an error I found this bug I resolved it with some conditional statement.
>* other than that I've updated from original website to current one
>* Actually after pulling changes I used npm install if any extra package is already installed when I was not working on this project, so I used npm install and run the project.
I think these are the things I've done

File: src/components/person_card/index.js (hunk 3 above)
Comment (thisisnitish) [reply src/components/person_card/index.js:49]: These are only auto indentation that's it nothing is changed, if u see again

**Comment by fahad-israr** [comment]
Expecting overriding of `package-lock.json` with `package.json` and merging.

**Pull request merged**